src/inmanta/data/__init__.py

```diff
@@ -239,9 +239,10 @@ def invert(self) -> "PagingOrder":
 
     @property
     def db_form(self) -> OrderStr:
+        # follow Postgres' default order for NULLS: matches default index order
         if self == PagingOrder.ASC:
-            return OrderStr("ASC NULLS FIRST")
-        return OrderStr("DESC NULLS LAST")
+            return OrderStr("ASC NULLS LAST")
+        return OrderStr("DESC NULLS FIRST")
```

Comment:
This has a significant impact on the performance, but it may be considered breaking? Do we dare make this change (for all API endpoints)? Or should we limit it to this one?